borg/helpers.py

```diff
@@ -8,6 +8,11 @@
 import os
 import pwd
 import re
+try:
+    from shutil import get_terminal_size
+except ImportError:
+    def get_terminal_size(fallback):
+        return (os.environ.get('COLUMNS', fallback[0]), os.environ.get('LINES', fallback[1]))
```

Comment:
maybe one could just copy most of the implementation from the py 3.3 stdlib and just fix it for 3.2.